packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -224,6 +226,12 @@ export class FilamentViewer extends LitElement {
 
     this[$updateSize]();
 
+    /*
+    this[$renderer].setClearOptions(
```

Comment:
i keep the comment here because this is the syntax to set clear color in 1.70 or higher, i will clean this in next pr where i update filament to latest version

Reply:
Ok, understood.